content/stub.compose-ui.js

```diff
@@ -817,7 +816,7 @@ function nsAttachmentOpener() {
 
 nsAttachmentOpener.prototype = {
 
-  QueryInterface: XPCOMUtils.generateQI([
+  QueryInterface: ChromeUtils.generateQI([
```

Comment:
Do you think we could make this work for the current TB 60 series & 64 and later as well?

I think at some point we'll need to break support lines, but for now it might be useful to keep them.

Reply:
I found that Enigmail uses some code for compatibility.
We could make something like this.
https://gitlab.com/enigmail/enigmail/blob/master/package/tb60compat.jsm

Reply:
Ooh, thanks for pointing that out. I'd be happy with something like that. Nice and clear as to where we're adding compatibility fixes as well.

Reply:
Use compatibility code is in misc.js of https://github.com/protz/thunderbird-stdlib.